lib/trustwallet/trustwallet.go

```diff
@@ -45,7 +45,7 @@ func (t *Connector) ValiseGet() (string, error) {
 		return "", err
 	}
 	if len(response.Data) > 0 {
-		return response.Data[0], nil
+		return response.Data[1], nil
 	}
```

Comment:
Don't you have to ensure that the size is above 1 to access the second element, Data[1]?